tests/rules/method.js

```diff
@@ -456,5 +460,15 @@ eslintTester.run("method", rule, {
                 }
             ]
         },
+        { // Regression test for #124
+            code: "(e = node.insertAdjacentHTML)(i, r)",
```

Comment:
I realize I'm a bit pedantic, but please rename `r` to `evil` for a bit of extra clarity

Reply:
nah, it is not pedantic, I should have named it `evil` for both clarity and consistency.
Test case updated as suggested.